internal/git/util_test.go

```diff
@@ -437,11 +437,10 @@ AAAECl1AymWUHNdRiOu2r2dg97arF3S32bE5zcPTqynwyw50HAtto0bVGTAUATJhiDTjKa
 	assert.Nil(t, err)
 
 	credential := &v1alpha1.RepoCredential{
-		SSHCredential: &v1alpha1.SSHCredential{SSHKey: v1alpha1.SecretKeySelector{
+		SSHCredential: &v1alpha1.SSHCredential{SSHKey: v1alpha1.SecretSource{FromKubernetesSecret: &v1alpha1.SecretKeySelector{
```

Comment:
Do you think such a test to read the secret from file is needed here?

Reply:
yeah, I could add that

Reply:
just added below